src/client/Microsoft.Identity.Client.Broker/RuntimeBroker.cs

```diff
@@ -94,10 +104,36 @@ private static void OnProcessExit(object sender, EventArgs e)
             }
         }
 
+        private void LogEventRaised(NativeInterop.Core sender, LogEventArgs args)
+        {
+            if (_logger != null)
+            {
+                using LogEventWrapper logEventWrapper = new LogEventWrapper(this);
+                LogLevel msalLogLevel = LogLevelMap[args.LogLevel];
+                if (_logger.IsLoggingEnabled(msalLogLevel))
+                {
+                    string msgWithPii = string.Empty;
+                    string msgNoPii = string.Empty;
+
+                    if (_logger.PiiLoggingEnabled)
```

Comment:
This is not how msal_runtime handles PII. They handle it by passing flag (in auth params?), in which case both exception emssages and log messages have PII.

Reply:
Pii will not be supported in this release. Created a feature request. https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3822